Packages/Env/Sources/Env/UserPreferences.swift

```diff
@@ -47,6 +47,8 @@ public class UserPreferences: ObservableObject {
   @AppStorage("swipeactions-icon-style") public var swipeActionsIconStyle: SwipeActionsIconStyle = .iconWithText
 
   @AppStorage("requested_review") public var requestedReview = false
+  
+  @AppStorage("collapse-long-posts") public var collapseLongPosts = true;
```

Comment:
; really? hahaha long time no see the semicolon

Reply:
shame on me. (Spring Data Elasticsearch is Java, in my business project we're using Kotlin). BUt you have them in the localization files as well!

Reply:
fixed